src/app/components/create-edit-event-page/create-edit-event-page.component.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../variables.scss';
+
+@mixin flex-container($flex-direction-value, $justify-content-value: center) {
+    display: flex;
+    flex-direction: $flex-direction-value;
+    justify-content: $justify-content-value;
+}
+
+h1 {
+    padding-left: 100px;
+    color: $yonder-blue;
+    font-weight: 500;
+}
```

Comment:
This should also be in style.scss,as h1 tag should have the same styling everywhere. If you need to change one of the values, you override that property for that specific component